Nuget.config

```
@@ -1,13 +1,13 @@
 <configuration>
 
   <packageSources>
-    <add key="github-lincoln" value="https://nuget.pkg.github.com/City-of-Lincoln-Council/index.json" />
+    <add key="github-hackney" value="https://nuget.pkg.github.com/LBHackney-IT/index.json" />
   </packageSources>
 
   <packageSourceCredentials>
     <github-lincoln>
-      <add key="Username" value="%LINCOLN_NUGET_USERNAME%" />
-      <add key="ClearTextPassword" value= "%LINCOLN_PACKAGES_TOKEN%" />
+      <add key="Username" value="PublicToken" />
+      <add key="ClearTextPassword" value= "%LBHPACKAGESTOKEN%" />
     </github-lincoln>
```

This would also need to change to hackney, 
```suggestion
    </github-hackney>
```